snp-pileup/tests/test-job-1.json

```diff
@@ -3,7 +3,7 @@
     "normal_bam": "input/test_N.bam",
     "dbsnp": "input/test.dbsnp_138.hg19.vcf",
     "expected_output": "expected/expected.bc.gz",
-    "publish_dir": "outdir",
+    "publish_dir": ".",
```

Comment:
I guess this is not intended? typically, when working on one package, we don't make change to files do not belong to the package. WFPM CLI currently does not enforce this rule, but I think it would be useful to implement it in WFPM CLI

Reply:
yes it wasn't intended - will change it back and push again.